tests/common/widgets/frequently-bought-together/options.ts

```diff
@@ -0,0 +1,429 @@
+            "currency": "EUR",
+            "discount_level": -100,
+            "discounted_value": 0,
+            "on_sales": false,
+            "value": 210
+          }
+        }
+              </li>
+              <li
+                class="ais-FrequentlyBoughtTogether-item"
+              >
+                {
+          "_highlightResult": {
+            "brand": {
+              "matchLevel": "none",
+              "matchedWords": [],
+              "value": "La Carrie Bag"
+            },
+            "name": {
+              "matchLevel": "none",
+              "matchedWords": [],
+              "value": "Bag La Carrie Bag small black"
+            }
+          },
+          "_score": 39.92,
+          "brand": "La Carrie Bag",
+          "list_categories": [
+            "Women",
+            "Bags",
+            "Shoulder bags"
+          ],
+          "name": "Bag La Carrie Bag small black",
+          "objectID": "A0E2000000024R1",
+          "parentID": "151",
+          "price": {
+            "currency": "EUR",
+            "discount_level": -100,
+            "discounted_value": 0,
+            "on_sales": false,
+            "value": 161.25
+          }
+        }
+              </li>
+            </ol>
+          </div>
+        </section>
+      `
+      );
+    });
+
+    test('renders transformed items', async () => {
+      const searchClient = createMockedSearchClient();
+
+      await setup({
+        instantSearchOptions: {
+          indexName: 'indexName',
+          searchClient,
+        },
+        widgetParams: {
+          objectIDs: ['objectID'],
+          transformItems(items) {
+            return items.map((item) => ({
+              objectID: item.objectID,
+              __position: item.__position,
+            }));
+          },
+        },
+      });
+
+      await act(async () => {
+        await wait(0);
+      });
+
+      expect(
+        document.querySelector('.ais-FrequentlyBoughtTogether')
+      ).toMatchNormalizedInlineSnapshot(
+        normalizeSnapshot,
+        `
+        <section
+          class="ais-FrequentlyBoughtTogether"
+        >
+          <div
+            class="ais-FrequentlyBoughtTogether-container"
+          >
+            <ol
+              class="ais-FrequentlyBoughtTogether-list"
+            >
+              <li
+                class="ais-FrequentlyBoughtTogether-item"
+              >
+                {
+          "objectID": "A0E200000002BLK"
+        }
+              </li>
+              <li
+                class="ais-FrequentlyBoughtTogether-item"
+              >
+                {
+          "objectID": "A0E200000001WFI"
+        }
+              </li>
+              <li
+                class="ais-FrequentlyBoughtTogether-item"
+              >
+                {
+          "objectID": "A0E2000000024R1"
+        }
+              </li>
+            </ol>
+          </div>
+        </section>
+      `
+      );
+    });
+
+    describe('components', () => {
```

Comment:
We shouldn't test this in the CTS since passing templates is flavor-specific.

Reply:
I mentioned in the description that this is a new behavior, because this is our first "true" unified ui component, which should not have differences between flavors as, except for the containing element (`rootTagName` in js), everything is coming from the ui component.